src/Client/Im/Main.purs

```diff
@@ -727,7 +727,8 @@ toggleConnectedWebSocket isConnected model@{ hasTriedToConnectYet, errorMessage
             else
                   [ pure $ Just UpdateDelivered ]
       where
-      lostConnectionMessage = "Connection to the server lost. Attempting to automatically reconnect..."
+      lostConnectionMessage = "Connection to the server lost. Attempting to automatically reconnect... \n\
```

Comment:
The new line doesn't actually work, lol.

Reply:
yes, it doesnt work in html